Source/Core/Core/PowerPC/PPCAnalyst.cpp

```diff
@@ -680,6 +687,7 @@ u32 PPCAnalyzer::Analyze(u32 address, CodeBlock *block, CodeBuffer *buffer, u32
 			code[i].branchTo = -1;
 			code[i].branchToIndex = -1;
 			code[i].skip = false;
+			code[i].isBranchTarget = HasOption(OPTION_FORWARD_JUMP) && branchTargets.count(address);
```

Comment:
If this is only looking for a count greater than zero, then shouldn't `find` just be used here, as it'll only look for the first occurrence, and not look for all of them.